service/confirm-reminder/src/helpers.ts

```diff
@@ -12,14 +12,12 @@ export const changeDate = (date: string, attempts: number, retryArray : number[]
   return new Date(oldDate.setDate(oldDate.getDate() + retryInterval)).toISOString();
 };
 
-export const retryExpired = (date: string, interval: number) => {
+export const retryValid = (date: string, interval: number) => {
   const today = new Date();
   today.setDate(today.getDate() - interval);
 
   const retryDate = new Date(date);
 
-  console.log("aa", date, today, retryDate, interval)
-
-  return retryDate < today;
+  return retryDate > today;
```

Comment:
not the opposite, it ">=" is the opposite of "<"

Reply:
Yes! Will fix